topic_change.py

```diff
@@ -173,7 +175,7 @@ def prepare_whitehall_csv(output_directory, source_topic, destination_topic):
               SOURCE=source_topic,
               DESTINATION=destination_topic,
               CSV_LOCATION=remote_path,
-    )
+              )
```

Comment:
Yes: `E124 closing bracket does not match visual indentation`

For what it's worth, vim also agrees with autopep8 as to the indentation. That's not to say we can't change it, but we'd need to tell flake to ignore it. I think it's better to leave as-is.